src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -101,8 +103,9 @@ public sealed partial class QuicStream : System.IO.Stream
         public override long Seek(long offset, System.IO.SeekOrigin origin) { throw null; }
         public override void SetLength(long value) { }
         public void Shutdown() { }
-        public System.Threading.Tasks.ValueTask ShutdownWriteCompleted(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public System.Threading.Tasks.ValueTask ShutdownCompleted(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public System.Threading.Tasks.ValueTask ShutdownWriteCompleted(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
not relevant to this PR, but... weren't we going to get rid of ShutdownWriteCompleted?

Reply:
I think it should have been resolved with the `DuplexStream` API, but that didn't pan out AFAIK. We should reopen the discussion about this.

cc: @scalablecory